build/Makefile

```diff
@@ -491,22 +507,37 @@ build-allocator-image: $(ensure-build-image) build-allocator-binary build-licens
 	docker build $(agones_path)/cmd/allocator/ --tag=$(allocator_tag) $(DOCKER_BUILD_ARGS)
 
 ifeq ($(WITH_WINDOWS), 1)
+ifeq ($(WITH_ARM64), 1)
 # push the gameservers sidecar image
-push-agones-sdk-image: push-agones-sdk-linux-image
+push-agones-sdk-image: push-agones-sdk-linux-image-arm64 push-agones-sdk-linux-image-amd64
+	docker image ls
```

Comment:
I'm guessing the `docker image ls` was left in by mistake 😄

Reply:
my bad